src/components/ChartContainer/D3Container.js

```diff
@@ -316,15 +333,35 @@ function D3Container({
               tableType="measure"
               rowData={MeasureTable.formatData(currentResults)}
               headerInfo={MeasureTable.headerData(isComposite)}
-              pageSize={MeasureTable.pageSize}
-              isComposite={isComposite}
               useCheckBox
               selectedRows={selectedMeasures}
               colorMapping={colorMap}
               handleTableFilterChange={handleTableFilterChange}
+              pageData={pageData}
             />
           )}
       </Grid>
+      <StyledEngineProvider injectFirst>
```

Comment:
This change, while totally works, makes me worry. Is `DisplayTable` a generic DataGrid like component to be used whenever we need a table? Or are we only using it for Measure and Patient data? You (Tim) didn't start this trend but if we continue down it the next time we need a DataGrid will will have to use MUI's DataGrid because DisplayTable will be so Measure/Patient specific it would be difficult to use it with anything else.

And I'm assuming we are planning on using the `DisplayTable` for the member info page? Will we now need to create and manage a new pagination class? Just something to think about.

Tim, I don't mean to rant at you. This is more a discussion for the whole team. **If we want to use in-house built components, I recommend we keep them generic and extensible. I'm worried we are going in the opposite direction with DisplayTable**

Back to just Tim, the way I would have done this is be putting a scrollable div inside the `DisplayTable`. That way the Pagination stays with `DisplayTable`
`<DisplayTable>
    <ScrollDiv>
        <Header>
        <Row>
        <Row>
    </ScrollDiv>
    <Pagination />
</DisplayTable>`